frontend/src/scenes/events/definitions/definitionDrawerLogic.ts

```diff
@@ -132,20 +179,51 @@ export const definitionDrawerLogic = kea<definitionDrawerLogicType<EventOrPropTy
         changeOwner: ({ ownerId }) => {
             actions.updateDefinition({ owner: ownerId })
         },
-        cancelDescription: () => {
-            actions.setDescription(values.definition?.description || '')
-            actions.setDescriptionEditing(false)
+        saveNewPropertyTag: ({ tag, currentTags, propertyId }) => {
+            if (currentTags?.includes(tag)) {
+                errorToast('Oops! This tag is already set', 'This event already includes the proposed tag.')
+                return
+            }
+            actions.setType('property_definitions')
+            actions.updateDefinition({ tags: [...(currentTags || []), tag] }, propertyId)
         },
-        saveDescription: () => {
-            actions.setDescriptionEditing(false)
-            actions.updateDefinition({ description: values.description })
+        deletePropertyTag: async ({ tag, currentTags, propertyId }, breakpoint) => {
+            await breakpoint(100)
+            actions.setType('property_definitions')
+            const tags = currentTags?.filter((_tag: string) => _tag !== tag)
+            actions.updateDefinition({ tags }, propertyId)
         },
-        updateDefinition: async ({ payload }) => {
+        updateDefinition: async ({ payload, id }) => {
             actions.setDefinitionLoading(true)
-            const response = await api.update(`api/projects/@current/${values.type}/${values.definition?.id}/`, payload)
-            actions.setDefinition(response)
+            const definitionId = id ? id : values.definition?.id
+            const response = await api.update(`api/projects/@current/${values.type}/${definitionId}/`, payload)
             actions.setDefinitionLoading(false)
-            eventDefinitionsModel.actions.setEventDefinitions(response)
+            if (values.type === 'event_definitions') {
+                actions.setDefinition(response)
+                eventDefinitionsModel.actions.setEventDefinitions(response)
+            } else {
+                actions.setPropertyDefinitions(response)
+            }
+        },
+        setPropertyDescription: ({ description, id }) => {
+            const prop = values.eventProperties.find((p) => p.id === id)
+            prop.description = description
+            actions.setDefinitionUpdateList(prop)
+        },
+        updateAllDescriptions: async () => {
+            actions.setSaveAllLoading(true)
```

Comment:
This `setSaveAllLoading` action could be removed and the `saveAllLoading` reducer could set itself to `true` when it sees the `updateAllDescriptions` action.

Not sure how to set it back to "false" though. Probably calling the action with `false` is fine in the end.